LeetCode/vDimple/String/String_Test/test.cpp

```diff
@@ -1,5 +1,61 @@
 #include "pch.h"
 #include"..\806 Number of Lines To Write String\806 NumberOfLines.h"
+#include"..\StringPermutations_D\StringPermutations.h"
+
+TEST(StringPermutations, Test1) {
+
+	std::string output;
+	testing::internal::CaptureStdout();
+	
+	std::string str="AABC";
+	
+	std::vector<std::string> res = getStringPermutations(str);
+	
+	for (auto each : res)
+		std::cout << each << std::endl;
+
+	output = testing::internal::GetCapturedStdout();
+
+	EXPECT_EQ(1, 1);
```

Comment:
Here we could equate the number of permutations for "AABC", which should be 12.

Reply:
incorporated this review comment.